packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/types/purchase_status.dart

```diff
@@ -26,4 +26,9 @@ enum PurchaseStatus {
   /// the purchase by calling the `completePurchase` method. More information on
   /// verifying purchases can be found [here](https://pub.dev/packages/in_app_purchase#restoring-previous-purchases).
   restored,
+
+  /// The purchase has been canceled.
+  ///
+  /// Update your UI to indicate the purchase is canceled.
+  canceled,
```

Comment:
The word `cancelled` should be with double "l":

```suggestion
  /// The purchase has been cancelled.
  ///
  /// Update your UI to indicate the purchase is cancelled.
  cancelled,
```